pkg/sdk/python/LICENSE.md

```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
```

Comment:
No need for this license, we already have a license in the top-level Mainflux repo